src/extra/others/controller/lv_obj_controller.c

```diff
@@ -0,0 +1,372 @@
+/**
+ * @file lv_obj_controller.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_obj_controller.h"
+
+#if LV_USE_OBJ_CONTROLLER
+
+#include "../../../extra/widgets/msgbox/lv_msgbox.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef struct manager_stack_t {
+    const lv_obj_controller_class_t * cls;
+    lv_obj_controller_t * controller;
+    bool obj_created;
+    bool destroying_obj;
+    bool dialog;
+    struct manager_stack_t * prev;
+} manager_stack_t;
+
+struct lv_controller_manager_t {
```

Comment:
In other parts of LVGL this convention is followed:
```c
struct _lv_controller_manager_t {
    lv_obj_t * container;
    lv_obj_controller_t * parent;
    manager_stack_t * top;
}lv_controller_manager_t ;
```

Please use it here too for consistency.

Reply:
Do you mean rename `struct lv_controller_manager_t` to `struct _lv_controller_manager_t` while keeping the typedef same? I have one question as clang-tidy complains about name starting with `_` is reserved.

Reply:
> Do you mean rename struct lv_controller_manager_t to struct _lv_controller_manager_t while keeping the typedef same? 

Yes!

> I have one question as clang-tidy complains about name starting with _ is reserved.

We violate this rule on other places too :slightly_smiling_face:

Reply:
I see. I'll rename this one.